### DISCUSSION TITLE
[Google Fonts] Will IP address of visitor be shared with google if we use rsms.me/inter CDN

### DISCUSSION THREAD

**Discussion by sahanaravi** [rsms/inter#610] [Q&A] [answered]
Saw that rsms.me/inter is distributed globally over CloudFlare's CDN, backed by GitHub's servers. Does this mean the visitor's IP will not be shared with google?I see that if the fonts library is not hosted locally on your domain, a connection is made to Google servers with every web visit and the IP address of your web visitor is shared with Google. Will it not be the case here?

**Comment by rsms** [comment]
> a connection is made to Google servers

Hm, that doesn’t sound right. How and where do you observe this? If you load inter.css in chrome and record using the network dev tool, can you share a screenshot showing the Google request? This should not even be technically possible.

**Reply by sahanaravi** [reply]
This is where I got it from https://developers.google.com/fonts/faq/privacy _. When end users visit a website that embeds Google Fonts, their browsers send HTTP requests to the Google Fonts Web API. The Google Fonts Web API serves the Google Fonts Cascading Style Sheets (CSS) and subsequently the font files specified in the CSS to the users. Such HTTP requests include (1) **the IP address used by the respective user to access the Internet,_**

**Reply by rsms** [reply, marked as answer]
You're confusing `rsms.me/inter` with Google Fonts. They are not the same. In fact, Inter from Google Fonts is a really old version, built by Google Fonts (not me.) The official fonts from `rsms.me/inter` are not affiliated with Google Fonts. They are served by a CDN provided by CloudFlare.

So simply use `https://rsms.me/inter/inter.css` (or download the fonts from [rsms.me/inter/download](https://rsms.me/inter/download/) and serve them yourself.)
